doc/components.md

```diff
@@ -63,6 +63,10 @@ This class represent a single object on the screen, being a floating rectangle o
 
 A `PositionComponent` has a `position`, `size` and `angle`, as well as some useful methods like `distance` and `angleBetween`.
 
+When implementing the `render` method for your component that extends `PositionComponent` remember to render from the top left corner (0.0).
+Your render method should not handle where on the screen your component should be rendered. To handle where and how your component should be rendered
```

Comment:
Not sure if I understood this sentence, maybe it would be easier just to say that the properties should be used on the render method?

Reply:
But the properties shouldn't be used in the render method, or not nececarily at least. This came from a discussion on Discord where someone rendered from the center, which obviosly messed with the anchor. And then I realized we didn't say anything about it in the docs.

Reply:
Oh, yes makes sense now, I think understood now what this means.

Reply:
Maybe we could mention the toRect methods? There is one that is a (0,0) rect and could be used, what do you think?

Reply:
I can add a paragraph about that too.

Reply:
Just something like that? (just update)

Reply:
Nice

Reply:
also mention that the reason for that is because you must call super.render, that is the one that does the translation. so if you forget (or forgo) calling super.render you will be rendered with no transforms

Reply:
Added a paragraph about that too now